Babylon/commands/api/scenarios/create.py

```diff
@@ -0,0 +1,51 @@
+import json
+from typing import Any
+
+from click import command, option
+
+from Babylon.commands.api.scenarios.service.api import ScenarioService
+from Babylon.utils.credentials import pass_azure_token
+from Babylon.utils.decorators import timing_decorator, wrapcontext, retrieve_state
+from Babylon.utils.response import CommandResponse
+
+payload = json.dumps({
+    "name": "Creating scenario with Babylon",
+    "description": "Brewery master reference analysis",
+    "tags": ["Brewery", "reference"],
+    "runTemplateId": "hundred",
+    "security": {
+        "default": "viewer",
+        "accessControlList": [{
+            "id": "elena.sasova@cosmotech.com",
+            "role": "admin"
+        }],
+    },
+})
+
+
+@command()
+@wrapcontext()
+@retrieve_state
+@pass_azure_token("csm_api")
+@timing_decorator
+@option("--organization_id", "organization_id", type=str)
+@option("--workspace_id", "workspace_id", type=str)
+def create(state: Any, organization_id: str, workspace_id: str, azure_token: str) -> CommandResponse:
+    """
+    Create new scenario
+    """
+    service_state = state["state"]
+    if organization_id:
+        service_state["api"]["organization_id"] = organization_id
+    if workspace_id:
+        service_state["api"]["workspace_id"] = workspace_id
+
+    # change this line when scenario manipulation is specified
+    spec = payload
```

Comment:
this should use data given in command call

Reply:
1. i thing it is correct, but you can simplify it:
`state["api"]["organization_id"] = organization_id or state['services']["api"]['organization_id"]`

2. change state['state'] to "state['services']